src/hearings/converters/reason-for-cancellation.answer.converter.ts

```diff
@@ -1,15 +1,25 @@
-import { Observable } from 'rxjs';
-import { map } from 'rxjs/operators';
-import { State } from '../store';
-import { AnswerConverter } from './answer.converter';
+import {ActivatedRoute} from '@angular/router';
+import {Observable} from 'rxjs';
+import {map} from 'rxjs/operators';
+import {LovRefDataModel} from '../models/lovRefData.model';
+import {State} from '../store';
+import {HearingsUtils} from '../utils/hearings.utils';
+import {AnswerConverter} from './answer.converter';
 
 export class ReasonForCancellationAnswerConverter implements AnswerConverter {
+  constructor(protected readonly route: ActivatedRoute) {
+  }
 
   public transformAnswer(hearingState$: Observable<State>): Observable<string> {
+    const cancelHearingActualReasons: LovRefDataModel[] = this.route.snapshot.data.cancelHearingActualReasons;
+
     return hearingState$.pipe(
       map(state => {
-        const hearingResponse = state.hearingRequest.hearingRequestMainModel.hearingResponse;
-        return hearingResponse && hearingResponse.hearingCancellationReason;
+        const hearingResultReasonType = state.hearingActuals.hearingActualsMainModel
```

Comment:
for better readability we ca create one variable to hold state.hearingActuals.hearingActualsMainModel and use that for all the conditions instead of having on each condition like
const hearingsMainModel = state.hearingActuals.hearingActualsMainModel;

Reply:
can't get what does this mean, as we passed the state so I assume this null check is unavoidable.